apps/src/lib/kits/maker/Thermometer.js

```diff
@@ -31,4 +32,60 @@ const Thermometer = {
     }
   }
 };
-export default Thermometer;
+export default PlaygroundThermometer;
+
+export class MicroBitThermometer extends EventEmitter {
+  constructor(board) {
+    super();
+    this.currentTemp = 0;
```

Comment:
Talked offline. This causes the initial reading to be 0C the first time the program is run after loading the page

Reply:
Coming back to this - this line isn't what is causing the initial reading to be 0C. this.currentTemp is only used to determine whether a data event can trigger the change event (i.e. has the temp changed from what it was before), but only the sensor value is ever returned from the thermometer. This issue is happening because analog channels are initialized to 0 and then we are reading the channel value before any data events have been emitted. So receiving the 0C here is behaving as expected (i.e. sensor hasn't received data, so it returns the initialized value).

tldr: Behavior is expected. No changes needed.

Reply:
Hmm that feels like a negative user experience - i.e. the first time I (or my friends/family/teacher) uses my app, it will always show 0C. Is there a way we can wait until the data event has been emitted to show temperature?

Reply:
Synced offline. Not needed to change this for the BETT demo. Erin's tracking this in follow-up work.

Reply:
Jira ticket to track^ https://codedotorg.atlassian.net/browse/STAR-936